src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -87,6 +87,16 @@
     <Compile Remove="runtime\PacketTracer\**\*.cs" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' And '$(_TargetFrameworkVersionWithoutV)' &lt; '5.0')">
+    <Compile Remove="libraries\System.Text.Json\Serializer\ReadPreservedReferences.cs" />
+    <Compile Remove="libraries\System.Text.Json\Serializer\WritePreservedReferences.cs" />
+  </ItemGroup>
+
+  <!-- TODO: Remove package dependency for future -->
```

Comment:
People that pull my changes will be broken and will need to install the latest SDK build; to avoid that, the 5.0 benchmarks  will use the NuGet package version of `System.Text.Json`.

Reply:
In general for tests targeting the current master branch we have the expectation that you need to always have an updated SDK matching it. It's fine to assume they'll have the right 5.0 bits.

Reply:
I see, then I will proceed to remove this. Thanks.

Reply:
> In general for tests targeting the current master branch we have the expectation that you need to always have an updated SDK matching it.

Interesting. I didn't know that. Sounds good :)

Does the `benchmarks_ci.py` script install the latest SDK for the developer?
https://github.com/dotnet/performance/blob/ca80d8e2886b583d0a69635740b188248d3d6fdd/docs/benchmarkdotnet.md#using-python-script